### PR TITLE
Added link for automated tests files in DOCUMENTATION_GUIDELINES.md

### LINKED ISSUE #992: Link for automated tests file broken in DOCUMENTATION_GUIDELINES.md
## Description
I clicked the [link](https://github.com/WordPress/openverse-api/blob/main/DOCUMENTATION_GUIDELINES.md#:~:text=Make%20sure%20the%20updates%20passed%20the%20automated%20tests%20in%20this%20file) for` automated tests file` in `DOCUMENTATION_GUIDELINES.md` and instead of going to the appropriate location is says `The 'WordPress/openverse-api' repository doesn't contain the '.github/workflows/integration-tests.yml' path in 'main'.`  I think the location of the file has been changed, If anyone could tell me the correct location I will be able to resolve this error.

## Reproduction

1. Go to [DOCUMENTATION_GUIDELINES.md](https://github.com/WordPress/openverse-api/blob/main/DOCUMENTATION_GUIDELINES.md#documentation-guidelines)
2. Navigate to [How to Start Contributing](https://github.com/WordPress/openverse-api/blob/main/DOCUMENTATION_GUIDELINES.md#how-to-start-contributing)
3. Click on the [link](https://github.com/WordPress/openverse-api/blob/main/.github/workflows/integration-tests.yml) in the 3rd point under [How to Start Contributing](https://github.com/WordPress/openverse-api/blob/main/DOCUMENTATION_GUIDELINES.md#how-to-start-contributing)
4. See error.

## Screenshots
image

## Environment

 - Device: Laptop
 - OS: Windows 10
 - Browser: Chrome
 - Version: 106.0.5249.119

## Resolution

- [x] 🙋 I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by Yash-Pratapwar**
## Fixes

Fixes #992 by @yash-pratapwar

## Description

This PR adds the link for automaed tests files in [DOCUMENTATION_GUIDELINES.md](https://github.com/WordPress/openverse-api/blob/main/DOCUMENTATION_GUIDELINES.md#documentation-guidelines)
## Testing Instructions

Go to [DOCUMENTATION_GUIDELINES.md](https://github.com/WordPress/openverse-api/blob/main/DOCUMENTATION_GUIDELINES.md#documentation-guidelines)
Navigate to [How to Start Contributing](https://github.com/WordPress/openverse-api/blob/main/DOCUMENTATION_GUIDELINES.md#how-to-start-contributing)
Click on the [link](https://github.com/WordPress/openverse-api/blob/main/.github/workflows/integration-tests.yml) in the 3rd point under [How to Start Contributing](https://github.com/WordPress/openverse-api/blob/main/DOCUMENTATION_GUIDELINES.md#how-to-start-contributing)
See the fix.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -142,3 +142,5 @@ ingestion_server/ingestion_server/lock
 .vscode
 
 openapi.yaml
+Pipfile
+Pipfile.lock
```
Comment (dhruvkb) [inline .gitignore:146]: `Pipfile` and `Pipfile.lock` must not be ignored.

**Review by dhruvkb — CHANGES_REQUESTED** [review]
The documentation change looks fine. Dependency management files shouldn't be ignored in VCS.

**Review by obulat — APPROVED** [review]
Thank you for updating the links, @Yash-Pratapwar! I'm approving on the condition that you fix the `.gitignore` file @dhruvkb mentioned.

File: .gitignore (hunk 1 above)
Comment (AetherUnbound) [reply .gitignore:146]: For more context, these two files are used for defining and locking our dependencies and are required for our project!

**Comment by Yash-Pratapwar** [comment]
Alright, will remove them from git ignore.

File: .gitignore (hunk 1 above)
Comment (zackkrida) [inline .gitignore:146]:
```suggestion
```

Comment (Yash-Pratapwar) [reply .gitignore]: Committed the changes.

**Review by AetherUnbound — APPROVED** [review]
Looks great! Thank you and congrats on your first contribution!

**Comment by Yash-Pratapwar** [comment]
@AetherUnbound Thank you!

**Pull request merged**